app/org/sagebionetworks/bridge/stormpath/StormpathDirectoryDao.java

```diff
@@ -201,7 +201,8 @@ private void adjustVerifyEmailPolicies(Study study, Directory directory) {
         ModeledEmailTemplate template = policy.getAccountVerificationEmailTemplates().single();
         updateTemplate(study, template, study.getVerifyEmailTemplate(), "verifyEmail");
 
-        policy.setVerificationEmailStatus(EmailStatus.ENABLED);
+        EmailStatus verifyEmailStatus = study.isEmailVerificationEnabled() ? EmailStatus.ENABLED : EmailStatus.DISABLED;
+        policy.setVerificationEmailStatus(verifyEmailStatus);
```

Comment:
Would it be better to have this test in StormpathDirectoryDaoTest, so that the code is 1:1 with the test?